week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -58,7 +76,36 @@ function exerciseTwo(shopping) {
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
 function exerciseThree(books) {
-  //Write your code in here
+    let content = document.querySelector("#content");
+    let ul = document.createElement("ul");
+    ul.setAttribute("class", "books");
+    content.appendChild(ul);
+
+    books.forEach((book) => {
+        let li = document.createElement("li");
+        let p = document.createElement("p");
+        let img = document.createElement("img");
+        img.setAttribute("width", "182");
+        img.setAttribute("height", "277");
+        p.innerText = book.title + " " + "-" + " " + book.author;
+        li.appendChild(p);
+        li.appendChild(img);
+        ul.appendChild(li);
+
+        if (book.title == "The Design of Everyday Things") {
+            img.src =
+                "https://images-na.ssl-images-amazon.com/images/I/41bWcNdTGmL._SX330_BO1,204,203,200_.jpg";
+            li.style.backgroundColor = "red";
```

Comment:
this definitely works perfectly. however, there is a more efficient way to do this by looking at the `alreadyRead` property in the `books` object